recipe/conda_build_config.yaml

```diff
@@ -13,21 +13,20 @@ zip_keys:
   - - macos_machine
     - cross_platform
     - uname_machine
-cxx_compiler:
-  - clang_bootstrap  # [osx]
-  - gxx              # [linux]
-  - vs2017           # [win]
+
 c_compiler:
   - clang_bootstrap  # [osx]
   - gcc              # [linux]
-  - vs2017           # [win]
-vc:
-  - 14
+cxx_compiler:
```

Comment:
```suggestion
cxx_compiler:  # [osx or linux]
```

Reply:
The whole build is skipped on windows, so this is kind of cosmetic, but sure.

Reply:
```suggestion
cxx_compiler:        # [osx or linux]
```